arm/Microsoft.ServiceBus/namespaces/deploy.bicep

```diff
@@ -200,6 +211,14 @@ module serviceBusNamespace_virtualNetworkRules 'virtualNetworkRules/deploy.bicep
   }
 }]
 
+module serviceBusNamespace_networkRuleSet 'networkRuleSets/deploy.bicep' = if (skuName == 'Premium' || !empty(networkRuleSets)) {
+  name: '${uniqueString(deployment().name, location)}-networkRuleSet'
+  params: {
+    namespaceName: serviceBusNamespace.name
+    networkRuleSet: networkRuleSets_var
```

Comment:
Please use separated parameters for child module properties and not the whole object to allow input validation. You could avoid building the var and use the single object properties here